tests/unit/src/BindingsTests.ts

```diff
@@ -0,0 +1,74 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { expect } from "chai";
+import { Container } from "inversify";
+import { it } from "mocha";
+
+import { Dependency, DependencyConfig } from "@itwin/cloud-agnostic-core";
+import { ServerStorageDependency } from "@itwin/object-storage-core";
+
+export interface DependencyBindingsTestCase {
+  symbolUnderTestName: string;
```

Comment:
This is an example, where modifier ends up in the middle and I'm not really sure what this really means anymore. Should this be something like `testedBindingsSymbol`, `testedBindingsName` or `testedClassIdentifier`? Symbol and string are very similar primative data types and `stringUnderTestName` would definitely sound wrong.

Reply:
This was also addressed in the previously merged PR: https://github.com/iTwin/object-storage/pull/47